src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -318,6 +319,23 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                             checkOverflow = false;
                             continue;
 
+                        case "nullable":
+                        case "nullable+":
+                            if (value != null)
+                            {
+                                break;
+                            }
+
+                            nullable = true;
+                            continue;
+
+                        case "nullable-":
+                            if (value != null)
+                                break;
```

Comment:
Nit: surround with braces to match file.

Reply:
> Nit: surround with braces to match file.

Matches the implementation for "checked" and I believe does not violate the style guidelines.

---
In reply to: [224930419](https://github.com/dotnet/roslyn/pull/30479#discussion_r224930419) [](ancestors = 224930419)